File: Java/1. INTRO POO/Producto.java
public class Producto {
	public String nombre;
	private String codigo;
	private double precio;

	public Producto() {
		nombre = "Sniker";
		codigo = "007";
		precio = 10.5;
	}

	public void mostrar() {
		System.out.println("Nombre: " + nombre);
		System.out.println("Codigo: " + codigo);
		System.out.println("Precio: " + precio);
	}
}

File: C#/1. INTRO POO/Producto.cs
using System;

namespace pruebita
{
	public class Producto
	{
		public string nombre;
		private string codigo;
		private double precio;
		public Producto(){
			nombre = "Sniker";
			codigo = "007";
			precio = 10.5;
		}
		public void mostrar(){
			Console.WriteLine("Nombre: "+nombre);
			Console.WriteLine("Codigo: "+codigo);
			Console.WriteLine("Precio: "+precio);
		}
	}
}
